feature/google/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt

```diff
@@ -697,11 +697,13 @@ class BillingWrapperTest {
         val type = BillingClient.SkuType.INAPP
         val time = System.currentTimeMillis()
         val sku = "sku"
-        val purchase = mockk<Purchase>().also {
-            every { it.purchaseToken } returns token
-            every { it.purchaseTime } returns time
-            every { it.sku } returns sku
-        }
+
+        val purchase = stubGooglePurchase(
+            purchaseToken = token,
+            purchaseTime = time,
+            productId = sku
+        )
+
```

Comment:
I am trying to move away from using these many mocks so I replaced with creating actual Purchases